mw-webapp/src/stories/Configure.mdx

```diff
@@ -0,0 +1,364 @@
+import { Meta } from "@storybook/blocks";
```

Comment:
We are not going to create separate stories without components in nearest future, so could we get rid of directory stories, looks like it will be empty?

Reply:
I was told to move Button into stories, will putt it back now

Reply:
Done

Reply:
Who and where told you to put it to the stories folder?

Reply:
Yeah, sorry for the misunderstanding, our fault. Thanks for notifying